# This contains the position of a point in 2D space
float64 x
float64 y